modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -80,6 +80,17 @@ def uploader
       @uploader ||= ClaimsApi::SupportingDocumentUploader.new(id)
     end
 
+    def flashes
+      initial_flashes = JSON.parse(to_internal).dig('form526', 'veteran', 'flashes')
+      homelessness = JSON.parse(to_internal).dig('form526', 'veteran', 'homelessness')
```

Comment:
If you could just add a spec test for this logic we should be in business, otherwise looks good.

Reply:
I just now saw your comment about spec tweaking